sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/receiver.py

```diff
@@ -67,15 +88,18 @@ def _incoming_transfer(self, frame):
             self._first_frame = frame
         if not self.received_delivery_id and not self._received_payload:
             pass  # TODO: delivery error
-        if self._received_payload or frame[5]:  # more
+        if self._received_payload or more:  # more
             self._received_payload.extend(frame[11])
-        if not frame[5]:
+        if not more:
             self._received_delivery_tags.add(self._first_frame[2])
             if self._received_payload:
                 message = decode_payload(memoryview(self._received_payload))
                 self._received_payload = bytearray()
             else:
-                message = decode_payload(frame[11])
+                try:
+                    message = decode_payload(frame[11])
+                except IndexError:
+                    message = decode_payload(b"")
```

Comment:
Not sure if this is the right way to handle an empty payload, but it seems like the best option. Thoughts?

Reply:
how does an empty payload come back today ?

Reply:
Outside of the amqpproxy, we haven't encountered an empty payload from the service afaik.